src/titiler/xarray/pyproject.toml

```
@@ -0,0 +1,70 @@
+[project]
+name = "titiler.xarray"
+description = "Xarray plugin for TiTiler."
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+    {name = "Vincent Sarago", email = "vincent@developmentseed.com"},
+    {name = "Aimee Barciauskas", email = "aimee@developmentseed.com"},
+]
+license = {text = "MIT"}
+keywords = [
+    "TiTiler",
+    "Xarray",
+    "Zarr",
+    "NetCDF",
+    "HDF",
+]
+classifiers = [
+    "Intended Audience :: Information Technology",
```

```suggestion
    "Development Status :: 4 - Beta",
    "Intended Audience :: Information Technology",
```
I think it would be useful to denote that this is less mature than the other titiler components in the project metadata